projects/1.16/assets/glassential/glassential/lang/zh_cn.json

```
@@ -1 +1,19 @@
-{}
\ No newline at end of file
+{
+    "block.glassential.glass_dark": "暗色玻璃",
+    "block.glassential.glass_dark_ethereal": "暗色空灵玻璃",
+    "block.glassential.glass_dark_ethereal_reverse": "反转暗色空灵玻璃",
+    "block.glassential.glass_ethereal": "空灵玻璃",
+    "block.glassential.glass_ethereal_reverse": "反转空灵玻璃",
+    "block.glassential.glass_ghostly": "幽灵玻璃",
+    "block.glassential.glass_light": "发光玻璃",
+    "block.glassential.glass_redstone": "红石玻璃",
+	
+    "tooltip.glassential.dark": "阻光",

```

```suggestion
    "tooltip.glassential.dark": "挡光",
```
https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/cc9c9dc025f0a45fa8b9bae637649108c75fb04b/projects/1.12.2/assets/1UNKNOWN/enderio/lang/zh_cn.lang#L695